ccpp/suites/suite_FV3_GFS_v17_coupled_p8_sfcocn.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<suite name="FV3_GFS_v17_coupled_p8_sfcocn" version="1">
```

Comment:
The RRTMGP in FV3_GFS_v17_coupled_p8 will be reverted back to RRTMG. Could you also use RRTMG for this SDF?

Reply:
@ChunxiZhang-NOAA Thanks. Do you want me to change now? or wait for RRTMG change.

Reply:
@uturuncoglu  Yes, please change it now. Please refer to: /scratch1/NCEPDEV/hwrf/save/Chunxi.Zhang/update_namelist_rt/ufs-weather-model/FV3/ccpp/suites/suite_FV3_GFS_v17_coupled_p8.xml

Reply:
@uturuncoglu Ah I see. Could you wait for one or two hours. The commit for these changes is almost done. You can pull this SDF from repos after the commit is done.

Reply:
@ChunxiZhang-NOAA Sure. Let me know when it is ready.

Reply:
@uturuncoglu The suite_FV3_GFS_v17_coupled_p8.xml was updated in the authoritative repos. Sorry for my late reply.

Reply:
@ChunxiZhang-NOAA I think I need another round of update of components one the top level UFS PR is ready. I have one issue in ORT threading test and I am expecting to solve it soon. Along the update process, I'll check the suite files and update the ones with sfc_ocean

Reply:
@ChunxiZhang-NOAA I updated the model based on head of develop and also update the suit file as you indicated.